common/src/main/java/org/apache/drill/common/DeferredException.java

```diff
@@ -123,6 +160,9 @@ public void suppressingClose(final AutoCloseable autoCloseable) {
         autoCloseable.close();
       } catch(final Exception e) {
         addException(e);
+      } catch(final Error e) {
+        logger.warn("caught Error {}", e);
```

Comment:
why log the error here ? we have a JIRA to try and do reduce error logging to the minimum [DRILL-2692](https://issues.apache.org/jira/browse/DRILL-2692)